hal/shared/platform_ncp.h

```diff
@@ -22,20 +22,28 @@
 #include "hal_platform.h"
 
 enum MeshNCPManufacturer {
-	MESH_NCP_MANUFACTURER_UNKNOWN,
-	MESH_NCP_MANUFACTURER_ESPRESSIF = 1,
-	MESH_NCP_MANUFACTURER_UBLOX = 2
+    MESH_NCP_MANUFACTURER_UNKNOWN,
```

Comment:
Since we've moved these definitions into a shared folder, let's rename all these enums to something more generic e.g. `PlatformNCPXxx` `PLATFORM_NCP_XXX` etc.

Reply:
Fixed